### DISCUSSION TITLE
"Right" way to build a production javascript bundle from gleam

### DISCUSSION THREAD

**Discussion by drewolson** [gleam-lang/gleam#3636] [Questions & support] [answered]
I'm curious if there's a built-in way to create a javascript bundle from gleam code. I wrote a [javascript example](https://github.com/drewolson/clip/tree/main/examples/javascript) for my library clip using `esbuild` and importing the generated javascript code in `build/dev` from my [entrypoint module](https://github.com/drewolson/clip/blob/main/examples/javascript/priv/cli.mjs), but I wasn't sure if this was the intended approach.

Thanks!

**Comment by lpil** [comment, marked as answer]
No official bundler recommendations, nope.
